02week/blog/css/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  font-family: 'Playfair Display', serif;
+  color: #365978;
+  background-color: #C2D652;
+}
+h1 {
+  font-size: 40px;
+  text-decoration: underline;
+  text-align: center;
+}
+.mainbox {
+  border: 20px;
+  border-style: solid;
+  border-color: #FF9933;
+  margin: 20px;
+  padding: 20px;
+}
+.article1 {
+  font-size: 20px;
+  color: #365978;
+}
+.head1 {
+  font-weight: bold;
```

Comment:
You have font-weight: bold; on lines 23, 30, and 39.  This is usually a clue that maybe you should have something like:  .fontBold {font-weight: bold;}